### PR TITLE
cloudutils: fix adding rocky9 host failure due to missing /etc/sysconfig/libvirtd

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes the issue while adding rocky9 host failure due to missing /etc/sysconfig/libvirtd

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

File: python/lib/cloudutils/configFileOps.py [hunk 1]
```diff
@@ -59,39 +60,42 @@ def getEntry(self, name, separator="="):
...
-            for entry in self.entries:
-                if entry.op == "add":
-                    if entry.separator == "=":
-                        matchString = "^\ *" + entry.name + ".*"
-                    elif entry.separator == " ":
-                        matchString = "^\ *" + entry.name + "\ *" + entry.value
-                else:
-                    if entry.separator == "=":
-                        matchString = "^\ *" + entry.name + "\ *=\ *" + entry.value
+        if os.path.exists(self.fileName) and os.path.isfile(self.fileName):
```
Comment (DaanHoogland) [inline python/lib/cloudutils/configFileOps.py:64]: So can we safely do without the libvirtd sysconfig file , @weizhouapache ?

Comment (weizhouapache) [reply python/lib/cloudutils/configFileOps.py:64]: yes, 
it will then create a new file `/etc/sysconfig/libvirtd` with content `LIBVIRTD_ARGS=-l`

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test rocky9 kvm-rocky9

**Comment by weizhouapache** [comment]
@blueorangutan test alma9 kvm-alma9

**Comment by weizhouapache** [comment]
@blueorangutan test rocky8 kvm-rocky8

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Review by shwstppr — APPROVED** [review]
Code LGTM. Test results also show no errors on Alma9

@weizhouapache currently we don't have support for Rocky9 on Trillian. Last time it was failing for me with some ansible version, syntax issue.
Oracle Linux 9 should work though

**Comment by weizhouapache** [comment]
> Code LGTM. Test results also show no errors on Alma9
> 
> @weizhouapache currently we don't have support for Rocky9 on Trillian. Last time it was failing for me with some ansible version, syntax issue. Oracle Linux 9 should work though

@shwstppr yeah, there could be some other issues while setting up a rocky9 environment.
Rohit has shared the steps in his testing: https://gist.github.com/rohityadavcloud/fc401a0fe8e8ea16b4b3a4e3d149ce0c
let's test it later

**Pull request merged**